content/spaces/backup-and-restore.md

```diff
@@ -133,30 +145,58 @@ metadata:
   name: my-awesome-ctp-backup
   namespace: default
 spec:
+  configRef:
+    kind: SharedBackupConfig
+    name: default
   controlPlane: my-awesome-ctp
   deletionPolicy: Delete
 ```
 
-Once your Space indicates the backup is complete, you can delete the managed control plane:
-
-```bash
-kubectl wait backup my-awesome-ctp-backup --for condition=Completed=True --timeout=3600s && \
-kubectl delete controlplane my-awesome-ctp
-```
-
 ### Restore
 
+
 <!-- vale off -->
-To restore from a backup, check the `ControlPlane` to make sure it's ready.
+You can restore a control plane's state from a backup in a few steps.
 <!-- vale on -->
 
+
+First, create a Secret in the source control plane:
```

Comment:
I would make it more explicit this is just to have some dummy data to backup and restore, it's not a required step.